src/pages/post-test-summary/post-test-summary.ts

```diff
@@ -188,4 +178,8 @@ export class PostTestSummaryPage {
       .filter((n) => n)
       .map((question: { id; keyWords }) => `${question.id} - ${question.keyWords}`);
   }
+
+  getTitle(): string {
+    return 'Test summary - Florence  Pearson';
```

Comment:
Can you confirm we can't get a quick win with a dynamic name here? If not then hardcoded is also fine.

Reply:
It would involve me passing the data through a number of screens that other people are currently working on. It would be a quick win but would cause conflicts on other peoples branches

Reply:
Yep, thought as much, no problem.